chihuahua/src/hcall/wasmtime.rs

```diff
@@ -527,6 +536,17 @@ impl Chihuahua for DummyWasmtimeHostProvisioningState {
             .add_new_data_source(metadata)
     }
 
+    #[inline]
```

Comment:
Missing Rustdoc comments, here and throughout.

Reply:
I put documentation in the first line of `impl Chihuahua ......` in a similar pattern as in the wasmtime.rs.